src/robotide/editor/macroeditors.py

```diff
@@ -85,17 +91,38 @@ def delete_rows(self):
     def delete(self):
         self.kweditor.OnDelete()
 
-    def comment(self):
+    def comment_rows(self):
         self.kweditor.OnCommentRows()
 
-    def uncomment(self):
+    def uncomment_rows(self):
         self.kweditor.OnUncommentRows()
 
+    def move_cursor_down(self):
```

Comment:
This function here makes a duplicate call. It was working without this here,
Same will happen with the create variables function.